src/common/libutil/grudgeset.c

```diff
@@ -0,0 +1,134 @@
+/************************************************************\
+ * Copyright 2021 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include <string.h>
+#include <errno.h>
+#include <czmq.h>
+
+#include <jansson.h>
+
+struct grudgeset {
+    json_t *set;
+    zhashx_t *grudges;
```

Comment:
SInce jansson is being used would it make sense for `set->grudges` to be a jansson object?

(File this under trivial, nitpicky, please ignore)

Reply:
I had it that way at first, but it seemed like it would be more code. I can try again.